FAQs/README.md

```diff
@@ -338,3 +341,50 @@ stages:
         azurePowerShellVersion: 'LatestVersion'
 ```
 
+- ### **UI**
+
+### **1. I am not able to see subscriptions in AzTS UI. Are there any prequisites or any settings that I am missing?**
+If you are not able to see subscriptions in AzTS UI , please check below prerequisites followed by troubleshooting steps:
+
+### Prerequisites:
+1. Signed in user must have one of the following permissions at subscription or resource group scope: Owner, Contributor, ServiceAdministrator, CoAdministrator, AccountAdministrator, Security Reader, Security Admin.
+
+2. Subscription scan should have completed for the day. Automated AzTS scans are configured to start at approximately 1:00 AM UTC. You can use the [On-Demand scan](../01-Setup%20and%20getting%20started/README.md#2-manually-trigger-azts-on-demand-scan-for-entire-tenant) command to trigger the scan immediately after the installation.
```

Comment:
To ensure the availability of data in AzTS, please ensure that the subscription scan has completed for the day. The automated AzTS scans are typically scheduled to start around 1:00 AM UTC. If you need to trigger a scan immediately after installing AzTS or at any other time, you can use the on-demand scan command to initiate the scan process right away.

Reply:
updated